tests/test-sources/plugins/by-name/faster/default.nix

```
@@ -0,0 +1,113 @@
+{
+  empty = {
+    plugins.faster.enable = true;
+  };
+
+  default = {
```

```suggestion
  defaults = {
```
nit (let's try to keep this consistent with other tests).